File: Easy/26. Remove Duplicates From Sorted Array.cpp
//C++

class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        for( auto x = (nums.begin()+1); x!=(nums.end()); x++)
        {
            if( *(x-1)==*(x))
            {
                nums.erase(x);
                x = nums.begin();
            }
        }
        int count=0;
        for( auto x:nums)
        {
            // cout<<x<<" ";
            count++;
        }
        return count;
    }
};

File: Easy/26. Remove Duplicates From Sorted Array.java
class Solution {
    public int removeDuplicates(int[] nums) { 
        int j = 1;
        int last = nums[0];
        for(int i=1; i<nums.length; i++){
           if(nums[i]!=last){
                nums[j]=nums[i];
                last=nums[i];
                j++;
           }
        }
        return j;
    }
}
